### DISCUSSION TITLE
Can I collaborate with the project?

### DISCUSSION THREAD

**Discussion by co-pas** [citizenfx/fivem#2689] [Engineering Discussion]
I'm trying to generate a build of the project, so I can understand it and start collaborating, but is that possible?

I get this:

C:\folder>git clone https://github.com/citizenfx/fivem.git -c core.symlinks=true
Cloning into 'fivem'...
remote: Enumerating objects: 119400, done.
remote: Counting objects: 100% (1386/1386), done.
remote: Compressing objects: 100% (696/696), done.
remote: Total 119400 (delta 720), reused 1304 (delta 663), pack-reused 118014
Receiving objects: 100% (119400/119400), 344.02 MiB | 4.91 MiB/s, done.
Resolving deltas: 100% (83595/83595), done.
error: unable to create symlink code/client/clrcore-v2/Client/FiveM/v1: Permission denied
error: unable to create symlink code/client/clrcore-v2/Math: Permission denied
error: unable to create symlink code/client/clrcore-v2/Server: Permission denied
error: unable to create symlink code/components/citizen-scripting-lua54/include: Permission denied
error: unable to create symlink code/components/citizen-scripting-lua54/src: Permission denied
error: unable to create symlink code/components/citizen-scripting-v8client/include: Permission denied
error: unable to create symlink code/components/citizen-scripting-v8client/src: Permission denied
error: unable to create symlink code/components/citizen-scripting-v8node/include: Permission denied
error: unable to create symlink code/components/citizen-scripting-v8node/src: Permission denied
error: unable to create symlink code/components/citizen-server-state-fivesv/include: Permission denied
error: unable to create symlink code/components/citizen-server-state-fivesv/src: Permission denied
error: unable to create symlink code/components/citizen-server-state-rdr3sv/include: Permission denied
error: unable to create symlink code/components/citizen-server-state-rdr3sv/src: Permission denied
error: unable to create symlink code/components/ros-patches-ny/include: Permission denied
error: unable to create symlink code/components/ros-patches-ny/lib: Permission denied
error: unable to create symlink code/components/ros-patches-ny/src: Permission denied
error: unable to create symlink code/components/ros-patches-rdr3/include: Permission denied
error: unable to create symlink code/components/ros-patches-rdr3/lib: Permission denied
error: unable to create symlink code/components/ros-patches-rdr3/src: Permission denied
error: unable to create symlink code/vendor/breakpad/src/third_party/lss: Permission denied
Updating files: 100% (11132/11132), done.
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

I followed the tutorial from there:

https://github.com/citizenfx/fivem/blob/master/docs/building.md

**Comment by spacevx** [comment]
Try to open cmd with admin rights
